packages/core/docs/commercetools/composables/use-facet.md

```diff
@@ -200,10 +196,48 @@ If the explicit configuration is not provided, the following defaults will be us
 }
 ```
 
-Configuration can be modified by passing identical configuration to:
+- `pageOptions` - an array of number of elements displayed per page.
+- `subcategoriesLimit` - the maximum number of subcategories displayed for any given category.
+- `availableFacets` - an array of filters available to the user.
+  - `facet` - facet expressions described on [this page](https://docs.commercetools.com/api/projects/products-search#termfacetexpression).
+  - `type` - `facet` data type. Valid values are `string`, `date`, `time`, `datetime`, `boolean` or `number`.
+  - `option` - filtering options described on [this page](https://docs.commercetools.com/api/projects/products-search#filters).
+  - `name` - facet alias described on [this page](https://docs.commercetools.com/api/projects/products-search#alias). `category` alias for the first facet shown above is a constant and shouldn't be changed.
+  - `filteringStrategy` - scope applied to this specific filter. Possible values are `filter`, `query` or `facets`. For more information refer to [this page](https://docs.commercetools.com/api/projects/products-search#filters). 
+- `sortingOptions` - an array of sorting options available to the user.
+  - `id` - unique `identifier` for the option.
+  - `name` - label for the option.
+  - `facet` - the name of the field to sort by. For more information refer to [this page](https://docs.commercetools.com/api/projects/products-search#sorting).
+  - `direction` - sorting direction. Valid values are `asc` or `desc`.
+- `filteringStrategy` - fallback scope applied to the facets that don't have strategy defined. Possible values are `filter`, `query` or `facets`. For more information refer to [this page](https://docs.commercetools.com/api/projects/products-search#filters).
+
+If the default configuration is modified, two identical copies must be passed to:
 - `@vsf-enterprise/ct-faceting/nuxt` module in `nuxt.config.js`.
 - `@vsf-enterprise/ct-faceting/server` integration in `middleware.config.js`.
 
+```javascript
+// nuxt.config.js
+export default {
+  buildModules: [
+    ['@vsf-enterprise/ct-faceting/nuxt', {
+      // options
+    }],
+  ]
+};
+
+// middleware.config.js
+module.exports = {
+  integrations: {
+    ctf: {
+      location: '@vsf-enterprise/ct-faceting/server',
+      configuration: {
+        // options
```

Comment:
What options? Shouldn't we provide an example with mocked values? It would be much simpler for developers, even for as after a break from this part :D

Reply:
I changed the order of code examples in this section. I show places where the configuration should be put, but still without any examples because by default no additional configuration is required.
Right after that, I show the defaults used when no configuration is provided, so the developer can only take pieces they want to modify.